app/api/localStorage/index.js

```diff
@@ -58,7 +60,7 @@ export default class LocalStorageApi {
 
   setTermsOfUseAcceptance = (): Promise<void> => new Promise((resolve, reject) => {
     try {
-      localStorage.setItem(storageKeys.TERMS_OF_USE_ACCEPTANCE, true);
+      localStorage.setItem(storageKeys.TERMS_OF_USE_ACCEPTANCE, JSON.stringify(true));
```

Comment:
😅

Reply:
yeah I don't know what to do about this one. Theoretically you can only save strings to localstorage (which is why the rest of this file uses `JSON` everywhere) but also saving non-strings seems to work perfectly fine

Reply:
<img width="577" alt="image" src="https://user-images.githubusercontent.com/2608559/57366138-6219de00-71c1-11e9-884a-e52afc0e5a9d.png">

It looks like Chrome just knows what to do with it. Even through `JSON.stringify(true)` returns "true" and not `true`, the result in `localstorage` is still just the literal `true` (like before) and not "true"

Reply:
yeah, I double checked and it works fine and we shouldn’t have any issues with compatibility. What scared me is that in the old code, it was just sent as a boolean and left to the API to decide to convert it. I think the change that you did is the *proper* way to do it.